lib/paleta/palette.rb

```diff
@@ -237,6 +209,25 @@ def fit
 
     private
 
+    def self.generate_from_image(path, size = 5)
+      unless Paleta.rmagick_available?
+        return puts "You must install RMagick to use Palette.generate(:from => :image, ...)"
```

Comment:
I'm thinking this should probably `raise` / `fail` as the method is completely unusable.